smart-contracts/tasks/origins/add.ts

```diff
@@ -287,7 +282,30 @@ task('pool:remove-origin', 'Removes an origin from a pool')
         process.exit()
       }
 
-      const tx = await pool.disableOrigin(l2ChainId, bridgeAddress)
-      console.log(`✅ Transaction sent! ${tx.hash}`)
+      if (origin.curator === userAddress) {
+        const tx = await pool.disableOrigin(l2ChainId, bridgeAddress)
+        console.log(`✅ Transaction sent! ${tx.hash}`)
+        return
+      }
+
+      // Else, let's check that maybe the curator is set?
+      const defaultCurator =
+        (await getOriginCuratorForNetwork(Number(chainId))) || userAddress
```

Comment:
```suggestion
        (await getOriginCuratorForNetwork(Number(chainId)))
```

that condition should not include the signer address as we are already checking for that above